site/app/views/admin/PlagiarismView.php

```diff
@@ -109,83 +105,71 @@ public function showPlagiarismResult($semester, $course, $gradeable_id, $gradeab
     public function configureGradeableForPlagiarismForm($new_or_edit, $gradeable_ids_titles, $prior_term_gradeables, $saved_config, $title) {
         $this->core->getOutput()->addBreadcrumb('Plagiarism Detection', $this->core->buildCourseUrl(['plagiarism']));
         $this->core->getOutput()->addBreadcrumb('Configure New Gradeable');
+        $this->core->getOutput()->addInternalCss("plagiarism.css");
+        $this->core->getOutput()->enableMobileViewport();
+
         $prior_term_gradeables_json = json_encode($prior_term_gradeables);
 
-        #default values for the form
+        // Default values for the form
         $gradeable_id = "";
-        $all_version = "checked";
-        $active_version = "";
-        $all_files = "checked";
-        $regex_matching_files = "";
+        $provided_code = false;
+        $provided_code_filenames = [];
+        $version = "all_versions";
         $regex = "";
-        $language = ["python" => "selected", "java" => "", "plaintext" => "", "cpp" => "", "mips" => ""];
-        $provided_code = "";
-        $no_provided_code = "checked";
-        $provided_code_filename = "";
+        $regex_dirs = ["submissions"];
+        $language = array_fill_keys(PlagiarismUtils::getSupportedLanguages(), "");
+        $language["plaintext"] = "selected";
```

Comment:
Doing this here, wouldn't this mean that `$language["plaintext"]` is always `selected` then as it doesn't look like this gets reset anywhere?